lib/cli/provider/index.js

```diff
@@ -154,6 +159,9 @@ module.exports.handler = function(argv) {
   , filter: function(device) {
       return argv.serial.length === 0 || argv.serial.indexOf(device.id) !== -1
     }
+    , exclude: function(device) {
```

Comment:
Yes I have no problem with adding the new option, however I would like the `filter` function to handle both cases.

Also I'm not 100% sure if I like that it's a pattern by default - consider, for example, a case where you have two devices with serials 012345678 and 01234567. Now the user tries to exclude only 01234567, but single it's a regex by default, both devices are excluded. Maybe I would like having both `--exclude` and `--exclude-pattern` (and possibly `--include` and `--include-pattern` as well).